### PR TITLE
[charts/portal] Additional labels

### PR DESCRIPTION
**Pull request by scottchc1**
**Description of the change**
additionalLabels for global and individual component. required for saas to apply custom labels from values.yaml.

**Benefits**
saas requirement

**Drawbacks**
none

**Applicable issues**
none

**Additional information**
all added variables are primarily for saas purpose, not displayed in values.yaml

**Checklist** 
- [x] Chart version bumped in `Chart.yaml` according to [semver](http://semver.org/).
- [x] Variables are documented in the README.md
- [x] Title of the PR starts with chart name (e.g. `[charts/gateway]`)
- [] If the chart contains a `values-production.yaml` apart from `values.yaml`, ensure that you implement the changes in both files

### CODE REVIEW

File: charts/portal/templates/jobs/cert-update-job.yaml [hunk 1]
```diff
@@ -11,6 +11,9 @@ metadata:
     chart: {{ template "portal.chart" . }}
     release: {{ .Release.Name }}
     heritage: {{ .Release.Service }}
+    {{- range $key, $val := .Values.global.additionalLabels }}
+    {{ $key }}: "{{ $val }}"
+    {{- end }}
```
Comment (scottchc1) [inline charts/portal/templates/jobs/cert-update-job.yaml:16]: note: all three jobs do not have their own values section, so not sure if it needs its own individual label

Comment (scottchc1) [reply charts/portal/templates/jobs/cert-update-job.yaml:16]: added as portal.jobs.additionalLabels

**Review by melil02 — APPROVED** [review]
LGTM

**Review by APIM-KiranVaddadi — COMMENTED** [review]
The labels are applied to top level recourses and not to the pods. If this the actual intent the changes looks good to me.

Can also be achieved with helm post render and Kustomize.

**Review by Gazza7205 — APPROVED** [review]
Looks good to me

**Comment by melil02** [comment]
> The labels are applied to top level recourses and not to the pods. If this the actual intent the changes looks good to me.
> 
> Can also be achieved with helm post render and Kustomize.

As per requirement it says labels should apply to Kubernetes workloads 😕

**Pull request merged**